deployments/azure-webapps-container.yml

```diff
@@ -11,7 +11,7 @@
 # 2. Create a secret in your repository named AZURE_WEBAPP_PUBLISH_PROFILE, paste the publish profile contents as the value of the secret.
 #    For instructions on obtaining the publish profile see: https://docs.microsoft.com/azure/app-service/deploy-github-actions#configure-the-github-secret
 #
-# 3. Create a GitHub Personal access token with "repo" and "read:packages" permissions. 
+# 3. Create a GitHub Personal access token with "repo" and "write:packages" permissions. 
```

Comment:
🤔 I don't see the personal access token being used below. Instead, github.token (which is functionally equivalent to the GITHUB_TOKEN secret, which is created for each workflow run) is being used. I think this line could be deleted fully.

Reply:
Ack I just noticed this isn't used in the workflow, it is used in the next set up step 🤦

Reply:
@anuragc617 I don't agree with that change on line 14. Here's why: 

- The `github.token` is used to push the container image to the GitHub Container Registry, so it needs `packages: 'write'`, which Anurag has in his PR. Perfect, that's great. Keep that.
- The PAT token is not used in the workflow. It is given on the Azure Web App to _pull_ the container from the registry, so it does **not** need write permissions. It only needs read permissions, as in my initial PR. Following the principal of least privilege, we shouldn't instruct users to create the PAT token with write permissions.  

Let's please remove the change on line 14 and merge this in so Sarah can merge the docs PR.

CC @skedwards88